packages/fxa-payments-server/.nsprc

```diff
@@ -1,3 +1,5 @@
 {
-  "exceptions": []
+  "exceptions": [
+    "https://npmjs.com/advisories/1184"
```

Comment:
Unrelated, but it came up today as a basically harmless advisory for our builds, so I figured I'd throw this in